AppCenterDataStorage/AppCenterDataStorageTests/NSObject+MSTestFixture.m

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "NSObject+MSTestFixture.h"
+
+@implementation NSObject (MSTestFixture)
+
+- (NSData *)getJsonFixture:(NSString *)fixture {
+  NSBundle *bundle = [NSBundle bundleForClass:[self class]];
+  NSString *path = [bundle pathForResource:fixture ofType:@"json"];
+  return [NSData dataWithContentsOfFile:path];
+}
+
+@end
```

Comment:
Shouldn't there be extra empty line in the end of file?

Reply:
There is in my local checkout...

If the formatter (which I ran) does not enforce that, I really don't mind.

Reply:
There is one in my local check out. I don't mind if the formatter does not enforce those.